isis/src/lro/apps/lronacpho/main.cpp

```diff
@@ -27,21 +29,24 @@ static bool g_useDEM;
 
 void phoCal (Buffer &in, Buffer &out);
 void phoCalWithBackplane (std::vector<Isis::Buffer *> &in, std::vector<Isis::Buffer *> &out );
+void GetCalibrationDirectory(QString calibrationType, QString &calibrationDirectory);
+
 /**
  *
  * @brief Photometric application for the LRO NAC cameras
  *
- * This application provides featurs that allow multiband cubes for LRO NAC cameras
+ * This application provides features that allow multiband cubes for LRO NAC cameras
  *   to be photometrically corrected
  *
  * @author 2016-09-16 Victor Silva
  *
  * @internal
- *   @history 2016-09-19 Victor silva - Adapted from lrowacpho written by Kris Becker
- *
+ *   @history 2016-09-19 Victor Silva - Adapted from lrowacpho written by Kris Becker
+ *	 @history 2021-03-12 Victor Silva - Updates include ability to run with default values
+ * 																			Added new values for 2019 version of LROC Empirical function.
  */
 void IsisMain (){
-  // Isis Processing by line
+  // Isis Processing by brick
```

Comment:
Why the change here? It's technically still true as ProcessByLine inherits from ProcessByBrick

Reply:
We were trying different ways to speed up processing in different scenarios and left this comment in erroneously. Thanks for the catch.